product-board/20230306.md

```diff
@@ -0,0 +1,50 @@
+# 2023-03-06
+## Participants
+
+* @lossack
+* @maxwolfs
+* @berendt
+* @garloff
+* @jschoone
+* @itrich
+* @fkr
+* @frosty-geek
+* @manuela-urban
+* @jnull
+
+## Agenda
+
+### @garloff to check currency of product-board mailing list
+_Hint: Managed by UCS group product-board_
+
+### How to move forward with OpenStack Health Monitor
+
+* VP12 needs to be filled with some details on where the future will take us
+* While we concluded in the past that something like CloudMon is the way to go, OpenStack Health Monitor is the project that still receives much more work and focus.
+* EOL openstack-health-monitor soon, stop adding features, shut down in next months, full focus on new cloud-mon
+* Kurt to take over cloud-mon
+* Knowledge transfer needed
```

Comment:
This does not cover the discussion imho.

I raised the point that we need to conclude HOW we're going to continue and wether this will be further work on OpenStack Health Monitor or something else, such as cloudmon.
And whatever we conclude there, needs to be the focus on VP12.
What I delegated back is the ownership of the overall topic (of monitoring the health of openstack cloud environments) not specifically cloudmon.

Reply:
This should be reflected better now.
If I still miss the point, please feel free to explain in ore detail or suggest better wording.